extensions/cornerstone/src/utils/measurementServiceMappings/utils/getSOPInstanceAttributes.js

```diff
@@ -13,8 +13,8 @@ import * as cornerstone from '@cornerstonejs/core';
  */
 export default function getSOPInstanceAttributes(
   imageId,
-  cornerstoneViewportService,
-  viewportId
+  cornerstoneViewportService = undefined,
```

Comment:
This seems unnecessary

Reply:
actually it was blocking building due to the typescript's wrong type error, as some of the code was calling the function with just one parameter.

All cornerstone tools' `function getMappedAnnotations(annotation, DisplaySetService)` function